nemo/collections/nlp/modules/common/megatron/parallel_adapters.py

```diff
@@ -54,9 +56,17 @@ def __init__(
         self.activation = activation_registry[activation]()
         self.norm_position = norm_position
 
-        self.linear_in = ColumnLinear(in_features, dim, bias=False, init_method=init_method_normal(0.2))
+        if column_init == 'xavier':
+            self.linear_in = ColumnLinear(in_features, dim, bias=False)
```

Comment:
Probably good to make all args within this adapter configurable? Such as bias and ColumnLinear init method as well?

Reply:
I'd leave bias as is, but init methods for row and column linear units are provided now.